cmd/soroban-rpc/internal/db/ledger.go

```diff
@@ -65,10 +68,39 @@ func (r ledgerReader) GetLedger(ctx context.Context, sequence uint32) (xdr.Ledge
 	case 1:
 		return results[0], true, nil
 	default:
-		return xdr.LedgerCloseMeta{}, false, fmt.Errorf("multiple lcm entries (%d) for sequence %d in table %q", len(results), sequence, ledgerCloseMetaTableName)
+		return xdr.LedgerCloseMeta{}, false, fmt.Errorf("multiple lcm entries (%d) for sequence %d in table %q",
+			len(results), sequence, ledgerCloseMetaTableName)
 	}
 }
 
+// GetLedgerRange pulls the min/max ledger sequence numbers from the meta table.
+func (r ledgerReader) GetLedgerRange(ctx context.Context) (ledgerbucketwindow.LedgerRange, error) {
+	var ledgerRange ledgerbucketwindow.LedgerRange
+
+	query := sq.Select("lcm.meta").
+		From(ledgerCloseMetaTableName + " as lcm").
+		Where(sq.Expr("lcm.sequence = (SELECT MIN(sequence) FROM " + ledgerCloseMetaTableName + ") " +
+			"OR lcm.sequence = (SELECT MAX(sequence) FROM " + ledgerCloseMetaTableName + ")"))
```

Comment:
Is this faster than using `IN()`?

Reply:
This seems to work:


```sql
select ledger_close_meta.* from ledger_close_meta, (select min(sequence) as min, max(sequence) as max from ledger_close_meta) minmax where ledger_close_meta.sequence in (minmax.min, minmax.max) ;
 ```

Reply:
BTW, this is not a blocked, but I would encourage you to try it out.

Reply:
Regardless, I think you can replace the Selects in `sq.Expr()` by actual `sq.Select()`

Reply:
Also, you can use `sq.Or` instead of a string.

Reply:
i.e. try to use `sq` as much as possible instead of text.

Reply:
Did you get to try using `IN` instead?

Reply:
I suspect it may be more performant since it uses a single subquery.

Reply:
@2opremio I think `IN` will complicate the query. squirrel does not have a built in `IN` function, and then we will have to add `UNION ALL` to combine both min and max queries for that.

Reply:
You can use IN in squirrel:

```go
stooges := users.Where(sq.Eq{"username": []string{"moe", "larry", "curly", "shemp"}})
three_stooges := stooges.Limit(3)
rows, err := three_stooges.RunWith(db).Query()

// Behaves like:
rows, err := db.Query("SELECT * FROM users WHERE username IN (?,?,?,?) LIMIT 3",
                      "moe", "larry", "curly", "shemp")
```

Reply:
Also, the query I suggested doesn't use `UNION ALL`

Reply:
Ahhh did not consider `sq.Eq`. That should work. I will try it out.